makefile

```diff
@@ -288,6 +293,7 @@ usbflash :
 	make -C tools/bootload_reboot_stub reboot
 	sleep 2.5
 	make -C tools/swadgeterm monitor
+#endif
```

Comment:
I think you got some C in your makefile, unless this was meant to be commented out?

Reply:
Nevermind, you fixed this. Dunno why github showed me this particular diff.